vendor/github.com/go-mysql-org/go-mysql/schema/schema.go

```diff
@@ -354,6 +354,8 @@ func (ta *Table) fetchIndexesViaSqlDB(conn *sql.DB) error {
 			&unused,
 			&unused,
 			&unused,
+			&unused,
+			&unused,
```

Comment:
What happened here? Did we update go.mod?

Reply:
This is something that needed an upstream fix https://github.com/go-mysql-org/go-mysql/pull/527. Since that has merged, I'll pull in those changes instead.

Reply:
This re-done and the updated version from https://github.com/go-mysql-org/go-mysql/pull/658 was pulled in instead.